flowcraft/templates/trimmomatic.py

```diff
@@ -368,12 +366,12 @@ def main(sample_id, fastq_pair, trim_range, trim_opts, phred, adapters_file,
         "MINLEN:{}".format(trim_opts[3]),
         "TOPHRED33",
         "-trimlog",
-        "{}_trimlog.txt".format(sample_id)
+        "/tmp/{}_trimlog.txt".format(sample_id)
     ]
 
     logger.debug("Running trimmomatic subprocess with command: {}".format(cli))
 
-    p = subprocess.Popen(cli, stdout=PIPE, stderr=PIPE)
+    p = subprocess.Popen(" ".join(cli), stdout=PIPE, stderr=PIPE, shell=True)
```

Comment:
Does it still required `shell=True` now that it's writing to `tmp`?

Reply:
indeed, it does not. I've altered this in the latest commit